uui/components/inputs/Slider.module.scss

```diff
@@ -81,39 +77,39 @@
         }
 
         :global(.uui-slider-scale-filled-dot) {
-            background-color: $blue;
-            border-color: $blue;
+            background-color: var(--uui-info-50);
```

Comment:
let's use primary color instead on info in all places

Reply:
fixed